test/core/vm/actor/builtin/v0/verified_registry/verified_registry_actor_test.cpp

```diff
@@ -0,0 +1,453 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "vm/actor/builtin/v0/verified_registry/verified_registry_actor.hpp"
+
+#include <gtest/gtest.h>
+
+#include "storage/ipfs/impl/in_memory_datastore.hpp"
+#include "testutil/mocks/vm/runtime/runtime_mock.hpp"
+#include "vm/actor/actor.hpp"
+#include "vm/state/impl/state_tree_impl.hpp"
+
+namespace VerifiedRegistry = fc::vm::actor::builtin::v0::verified_registry;
+using fc::primitives::address::Address;
+using fc::storage::ipfs::InMemoryDatastore;
+using fc::vm::VMAbortExitCode;
+using fc::vm::VMExitCode;
+using fc::vm::actor::kStorageMarketAddress;
+using fc::vm::actor::kSystemActorAddress;
+using fc::vm::runtime::MockRuntime;
+using fc::vm::state::StateTreeImpl;
+using testing::Return;
+using VerifiedRegistry::DataCap;
+using VerifiedRegistry::kMinVerifiedDealSize;
+using VerifiedRegistry::StoragePower;
+
+struct VerifiedRegistryActorTest : testing::Test {
```

Comment:
I suggest using ActorTestFixture from https://github.com/filecoin-project/cpp-filecoin/pull/324/files#diff-2a1e8ea04a306a54d0428e4e40f846abe3d724f7d71b2b16b79a7ead9652e22b
and remove StateTreeImpl dependency from all actor tests in this PR.

Reply:
At least see https://github.com/filecoin-project/cpp-filecoin/tree/feature/verified-registry-actor-tests

Reply:
Waiting for your merge

Reply:
#324 merged

Reply:
Done